Week-1/exercise/index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- styles -->
+    <link rel="stylesheet" href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css">
+    <link rel="stylesheet" href="css/styles.css">    
+
+    <title>Document</title>
+</head>
+<body>
+    <div>
+        <nav class="sidebar">
+            <i class="las la-bars"> </i>
+            <i class="las la-home"> </i>
+            <i class="las la-search"></i>
+            <i class="las la-volume-up"></i>
+            <i class="las la-user"></i>
+            <i class="las la-cog"></i>
+            <i class="las la-spotify"></i>
+            <i class="las la-soundcloud"></i>
+            
+    
+        </nav>
+        <header>
+            <div>
+                <img src="https://uploads.codesandbox.io/uploads/user/a8233562-b36b-483d-9de9-5a1ac148fb0c/2Eq--cattyboard.jpg" alt="image" class="header-img">
```

Comment:
Give proper Alternate text for the images

Reply:
noted